src/js/client/renderer.js

```diff
@@ -97,7 +98,13 @@
             }
         },
         events: {
-            "onTemplatesLoaded": null
+            "onTemplatesLoaded": null,
+            "onAllResourcesLoaded": {
+                events: {
+                    // onMessagesLoaded: "onMessagesLoaded",
```

Comment:
You need to explain why you commented `onMessagesLoaded` out.  If there' s a new mechanism that ensures that rendering can take place after messages are loaded, fine, then it should be removed.  If it's here for some other reason, you need a TODO or comment.